app/src/renderer/vuex/modules/blockchain.js

```diff
@@ -85,11 +109,21 @@ export default ({ node }) => {
       commit(`setSyncing`, false)
 
       // only subscribe if the node is not catching up anymore
```

Comment:
is this comment true? we used to run into an issue where voyager would connect to nodes that weren't caught up yet. we don't have this problem right now because we're using hosted nodes... so we should implement this now to accommodate for syncing nodes or delete the comment and wait until it's an issue later.

Reply:
I was thinking the same, but if our stargate went off for a while is probably wise to let him catch up before reading blocks. But once we remove completely the RPC then we have to move this logic to stargate itself (if we want to keep it)

Reply:
sure ok. that makes sense. but what i was trying to say is that we're not checking if the node is syncing or not... are we?